hadoop-ozone/dist/src/main/smoketest/repair/om-compact.robot

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Test for OM DB Compaction Repair Tool
+Library             OperatingSystem
+Library             BuiltIn
+Resource            ../commonlib.robot
+Test Timeout        10 minutes
+
+*** Variables ***
+${OM_DB_PATH}       /data/metadata/om.db
+${KEY_COUNT}        200000
+${THREAD_COUNT}     20
+
+*** Keywords ***
+Create Test Keys
+    Execute    ozone freon ockg -n ${KEY_COUNT} -t ${THREAD_COUNT} -s 0
+
+Delete Test Keys
+    Execute    ozone fs -rm -R -skipTrash ofs://omservice/vol1/bucket1
+
+Get OM DB SST Files Size
+    ${output} =    Execute    find ${OM_DB_PATH} -name '*.sst' -exec du -b {} + | awk '{sum += $1} END {print sum}'
+    ${sst_size} =  Evaluate   int("${output}".strip())
```

Comment:
nit: `Convert To Integer` may be easier

https://robotframework.org/robotframework/6.1.1/libraries/BuiltIn.html#Convert%20To%20Integer